### PR TITLE
Add Record Type node to Monument and Area graphs

### LINKED ISSUE #1227: Record Type node in Mons and Areas
Add a "Record Type" node in the Monument and Areas models with datatype domain-value and values of "Monument" and "Building".

### PR DESCRIPTION
**Pull request by samuelscandrett**
#1227 

Adds a "Record Type" node in the Monument and Areas models with datatype domain-value and values of "Monument" and "Building".

image

### CODE REVIEW

**Comment by aj-he** [comment]
Confirmed with @SDScandrettKint that this does not need a report change.

File: arches_her/pkg/graphs/branches/Record Type.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
+{
+    "graph": [
+        {
+            "author": "",
```
Comment (csmith-he) [inline arches_her/pkg/graphs/branches/Record Type.json]: You need to add an author for this branch - preferably the author's name and organisation
(e.g. we have Philip Carlisle - Historic England for the ones Phil has created)

Comment (samuelscandrett) [reply arches_her/pkg/graphs/branches/Record Type.json]: Added in latest commit https://github.com/archesproject/arches-her/pull/1228/commits/3bcf0379505261a4a8fceaa2405365032f48a845

**Review by csmith-he — APPROVED** [review]
All looks good to me and no issues spotted when spinning this up in an instance of AfHER

**Pull request merged**